DirectXTex/Filters.h

```diff
@@ -195,9 +195,9 @@ inline void _CreateCubicFilter(_In_ size_t source, _In_ size_t dest, _In_ bool w
     XMVECTOR d0 = (p0) - a0; \
     XMVECTOR d2 = (p2) - a0; \
     XMVECTOR d3 = (p3) - a0; \
-    XMVECTOR a1 = d2 - g_cubicThird*d0 - g_cubicSixth*d3; \
-    XMVECTOR a2 = g_cubicHalf*d0 + g_cubicHalf*d2; \
-    XMVECTOR a3 = g_cubicSixth*d3 - g_cubicSixth*d0 - g_cubicHalf*d2;  \
+    XMVECTOR a1 = d2 - XMVectorMultiply( g_cubicThird, d0 ) - XMVectorMultiply( g_cubicSixth, d3 ); \
```

Comment:
Why didn't you have to use XMVectorSubtract( here? How does operator-(FXVMECTOR, FXMVECTOR) work here?

Reply:
All the code around diffuse-errors also needs ``operator +=(FMXVECTOR)`` which also fails to build.

Reply:
In ``filters.h``, the macros ``BILINEAR_INTERPOLATE`` and ``TRILINEAR_INTERPOLATE`` also fail to build without the overloads.

Reply:
It's strange I succeed to compile, what toolset are you using ?